prow/config.yaml

```diff
@@ -1492,6 +1492,38 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  - name: pull-kubernetes-e2e-kubeadm-dind-ipv6
+    agent: kubernetes
+    context: pull-kubernetes-e2e-kubeadm-dind-ipv6
+    max_concurrency: 8
+    skip_report: true
+    run_if_changed: '^(cmd/kubeadm|build/debs).*$'
+    rerun_command: "/test pull-kubernetes-e2e-kubeadm-dind-ipv6"
+    trigger: "(?m)^/test pull-kubernetes-e2e-kubeadm-dind-ipv6,?(\\s+|$)"
+    skip_branches:
+    - release-1.6 # doesn't support IPv6
```

Comment:
we no longer support testing these branches (< 1.8) at all so we can probably drop this. we only support the officially supported versions + under development version (1.9, 1.10, 1.11; master/1.12)

Reply:
Will remove the skip_branches.